hedera-node/src/main/java/com/hedera/services/contracts/execution/EvmTxProcessor.java

```diff
@@ -185,10 +188,11 @@ protected TransactionProcessingResult execute(Account sender, Address receiver,
 		Set<Address> addressList = new HashSet<>();
 		Multimap<Address, Bytes32> storageList = HashMultimap.create();
 		int accessListStorageCount = 0;
```

Comment:
nit: Instead of declaring the variable here, why don't we return it as part of the `processContractAccessList` method?